### ISSUE TITLE
output from `apoc.export.cypher.all` on Russian Twitter Trolls data set causes error when run

### ISSUE THREAD

**Issue by eastlondoner** [neo4j-contrib/neo4j-apoc-procedures#1286]
## Expected Behavior (Mandatory)

When I run the cypher from `apoc.export.cypher.all` it should complete without error

```
# This should complete without error:
cat import/twitter.cypher | ./bin/cypher-shell --non-interactive
```

## Actual Behavior (Mandatory)

SyntaxException
```
cat import/twitter.cypher | ./bin/cypher-shell --non-interactive
org.neo4j.cypher.internal.v3_5.util.SyntaxException: Invalid input '}': expected whitespace, comment or a property key name (line 1, column 679 (offset: 678))
```

## How to Reproduce the Problem

Use the Russian Twitter Trolls Sandbox data (https://neo4j-sandbox-usecase-datastores.s3.amazonaws.com/v3_4/twitter-trolls.db.zip). 

Create export cypher using `call apoc.export.cypher.all('twitter.cypher',{format:'cypher-shell', useOptimizations: {type: 'unwind_batch_params', unwindBatchSize:100}})`

Get a new, empty neo4j instance and execute the exported cypher against it e.g.
`cat import/twitter.cypher | ./bin/cypher-shell -u neo4j -p <your password> --non-interactive`

TLDR; The problematic lines are somewhere in this section [problem2.txt](https://github.com/neo4j-contrib/neo4j-apoc-procedures/files/3602299/problem2.txt)

### Versions
  - Neo4j: 3.5.8
  - Neo4j-Apoc: 3.5.0.4

**Comment by eastlondoner** [comment]
I built the latest apocalypse 3.5 jar from source and tried again based on a tip off from @jexp.
I now get a different error during import:

```
org.neo4j.driver.v1.exceptions.ClientException: Invalid input ':': expected <init> (line 1, column 1 (offset: 0))
```

The problematic cypher is in these 3 lines:
[problem3.txt](https://github.com/neo4j-contrib/neo4j-apoc-procedures/files/3602469/problem3.txt)

**Comment by eastlondoner** [comment]
So I can use this workaround `cat export.cypher | sed $'s/\xc2\x85/\\\\n/g' | cypher-shell ...` to replace the offending unicode characters with newlines - but technically that's actually changed the content. I _hope_ that there is a way to escape the unicode character with cypher shell

**Comment by conker84** [comment]
Cool thanks for the hint!
